### ISSUE TITLE
background colour outside the content area

### ISSUE THREAD

**Issue by brilliantweb** [zurb/foundation-emails#136]
Hi

Seriously impressed with INK.
Know just enough to know that I mustn't mess (too much) with the code.

I want a background colour down the left and right side of the email.
This is so the content of the email occupies a well defined space on a desktop mail client.

I tried setting the body background colour.
The problem is that the main body table cell has no padding, so the text is flush up against the edge of the background.

How can I adjust the content without breaking the super-tricky compatibility with multiple email clients?

Thanks
Brendon

**Comment by freemagee** [comment]
Hi,

I have achieved something similar I believe to what you want. I added a class to the table cell that contains all the content called 'email-pad'. It adds 10px padding left and right and a white bg.

I experimented with wrapping everything with a simple new table, but it ends up breaking the INK code. This was my solution to stay within the INK sandbox.

See screenshot:

![bg-colour-eg](https://cloud.githubusercontent.com/assets/1529405/9194931/e6483e96-4016-11e5-94e2-6dc0bde3bcc1.jpg)
